fixieai/agents/metadata.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Literal, Optional, Union
+
+from pydantic import dataclasses as pydantic_dataclasses
+
+from fixieai.agents import corpora as fixie_corpora
+from fixieai.agents import llm_settings
+
+
+@pydantic_dataclasses.dataclass
+class StandaloneAgentMetadata:
```

Comment:
should these live within the respective agent files?

Reply:
I put them here to avoid circular imports because the abstract method returns `Union[Standalone, CodeShot]`.